packages/eslint-plugin/package.json

```diff
@@ -21,6 +21,7 @@
     "test:watch": "glob './src/**/__tests__/*.test.ts' --cmd='node --loader tsx --no-warnings --test --watch'"
   },
   "dependencies": {
+    "@typescript-eslint/utils": "^5.60.1",
```

Comment:
The `✅ Check` CI workflow will fail because this isn't pinned; needs a yarn install too

```suggestion
    "@typescript-eslint/utils": "5.60.1",
```

Reply:
🙏